src/bp/core/services/action/local-action-server.ts

```diff
@@ -71,7 +71,7 @@ export class LocalActionServer {
   }
 
   public listen() {
-    const port = process.env.PORT
+    const port = process.env.BOTPRESS_PORT || process.env.PORT
```

Comment:
The local action server is a sub process, and the port is not the botpress_port, it is provided here https://github.com/botpress/botpress/blob/master/src/bp/core/botpress.ts#L194